daemon/cmd/daemon_main.go

```diff
@@ -1540,6 +1540,14 @@ func initEnv(vp *viper.Viper) {
 			)
 		}
 	}
+
+	// Ensure the xdp-ignore-device-regex compiles
+	if len(option.Config.XDPIgnoreDeviceNameRegex) > 0 {
+		_, err := regexp.Compile(option.Config.XDPIgnoreDeviceNameRegex)
+		if err != nil {
```

Comment:
Tip: this can be folded like `if _, err := regexp.Compile(option.Config.XDPIgnoreDeviceNameRegex); err != nil { ... }`

Reply:
TIL - thank you!